src/sections/Meshmap/Meshmap-design/meshmap-design-banner.js

```diff
@@ -1,11 +1,12 @@
 import React from "react";
 import styled from "styled-components";
-import { ReactComponent as BannerImage } from "./images/banner-image-colorMode.svg";
-import { ReactComponent as CytoscapeImg } from "./images/cytoscape-colorMode.svg";
-import CytoscapeDemo from "./cytoscape-demo";
-import CytoscapeCanvas from "./cytoscape-demo";
-import Graph from "./cytoscapeCanvas";
-import CytoscapeComponent from "./cytoscapeCanvas";
+// uncomment if need
+// import { ReactComponent as BannerImage } from "./images/banner-image-colorMode.svg";
+// import { ReactComponent as CytoscapeImg } from "./images/cytoscape-colorMode.svg";
+// import CytoscapeDemo from "./cytoscape-demo";
+// import CytoscapeCanvas from "./cytoscape-demo";
+// import Graph from "./cytoscapeCanvas";
+// import CytoscapeComponent from "./cytoscapeCanvas";
```

Comment:
Curious to know why we have left this uncommented? Any particular reason?

Reply:
It's because these were unused. These import statements may as well be removed.

Reply:
Yes as a good practice eslint wouldn't allow us that's why i did this let me know if any thing is wrong or i need to revert this changes

Reply:
Cool, thanks for clearing it @Shivam-AfA 😄

Reply:
I think we should remove unused imports.

Reply:
Yes because unused import just increases the load and it not healthy for website @samyakshah3008